nodejs/lib/yamlscript/index.js

```diff
@@ -0,0 +1,92 @@
+const ffi = require('ffi-napi');
+const ref = require('ref-napi');
```

Comment:
ugh, I tried like 4 ffi libs including this one, another inactive one, and 2 actively maintained ones, and I just kept getting segfaults.

In all cases it looked like the first graalvm ffi call to create the isolate didn't segfault (tho maybe I did it wrong anyways, causing subsequent uses of that isolate to segfault), but the `libyamlscript` and graalvm teardown ffi calls segfaulted.

Was that the case with this implementation?

Reply:
iirc I was able to successfully create and teardown the isolate.
I want to try calling a function with a Long or Double arg and return value instead of a string buffer.

Reply:
yeah good idea, lmk how that goes!!!